src/app/qan/profile-table/profile-table.component.ts

```diff
@@ -213,7 +213,7 @@ export class ProfileTableComponent implements OnInit, OnDestroy, AfterViewInit {
     };
     const dimension = dimensionsMap[this.currentParams.group_by] ?
       dimensionsMap[this.currentParams.group_by] : `${GroupByMock[this.currentParams.group_by]}s`;
-    return `${this.tableData['total_rows']} distinct ${dimension}`;
+    return `${this.paginationConfig.totalItems} distinct ${dimension}`;
```

Comment:
`this.currentParams.group_by` goes against DRY
Please prefer creating a variable if you access it multiple times, i.e. 
`const { group_by } = this.currentParams`;